picasso/src/test/java/com/squareup/picasso/RequestCreatorTest.java

```diff
@@ -555,6 +565,11 @@ public void intoTargetNoResizeWithCenterInsideOrCenterCropThrows() {
       fail("Center inside with unknown height should throw exception.");
     } catch (IllegalStateException ignored) {
     }
+    try {
+      new RequestCreator(picasso, URI_1, 0).onlyScaleDown().into(mockTarget());
+      fail("onlyScaleDown with unknown height should throw exception.");
```

Comment:
Shouldn't this fail when `.onlyScaleDown()` is called without a call to `.fit()` or `.resize(int, int)`?